paasta_tools/vitesscluster_tools.py

```diff
@@ -287,6 +289,28 @@ def get_cell_config(
                 "mysql_auth_vault_tls_ca": f"/etc/vault/all_cas/acm-privateca-{region}.crt",
                 "mysql_auth_vault_ttl": "60s",
             },
+            extraVolumeMounts=[
```

Comment:
should we mount the default volumes defined in SystemPaastaConfig? or are we sure we only need/want srv-configs? (i.e., this isn't mounting the /nail/etc/{habitat,runtimeenv,superregion,etc} files)

Reply:
@nemacysts This is following the same pattern suggested earlier for vttablet volume mounts https://github.com/Yelp/paasta/blob/master/paasta_tools/vitesscluster_tools.py#L477-L516 . If there is an alternate/recommended way to do this?

For vtgates, we need the /nail/srv/configs/dre_service_discovery.yaml file as a part of lifecycle hook scripts

> i.e., this isn't mounting the /nail/etc/{habitat,runtimeenv,superregion,etc} files
Wouldn't the volumes mounted from the kube machine host paths be already having habit/runtimeenv/superregion specific srv-configs?

Reply:
@VinaySagarGonabavi `system_paasta_config.get_volumes()`will read `/etc/paasta/volumes.json` and return some default mounts that are usually needed/helpful to have

> Wouldn't the volumes mounted from the kube machine host paths be already having habit/runtimeenv/superregion specific srv-configs?
yes, this is unrelated to srv-configs -  having these files inside your pods would allow for things like the register/deregister scripts to discover where a pod is running rather than having to configure those lifecycle hooks region parameters inside paasta :)

Reply:
I'll update the script to find the region based on where it's running. The reliance on srv-configs is to find the cloudmap service id stored in srv-configs https://github.yelpcorp.com/sysgit/srv-configs/blob/master/ecosystem/devc/dre_service_discovery.yaml#L23

Reply:
++. Let me also add `system_paasta_config.get_volumes()` so we can find /nail/etc/region etc.,

Reply:
@nemacysts https://github.com/Yelp/paasta/pull/3959